review.md

```diff
@@ -2,48 +2,72 @@
+
+and if the [pull request](https://github.com/r-multiverse/contributions/pulls) itself:
+
+1. Was [created by the GitHub web interface](contributors.md).
+1. Adds new contributed listings to the [`packages` folder](https://github.com/r-multiverse/contributions/tree/main/packages).
+1. Does not add, modify, or delete any other files in <https://github.com/r-multiverse/contributions>.
+
+and if each new [contributed listing](https://github.com/r-multiverse/contributions/tree/main/packages):
+
+1. Is a single line of text with a valid HTTPS URL.
+1. Points to an existing public GitHub/GitLab repository.
+
+and if the contributed GitHub/GitLab repository:
+
+1. Includes a GitHub/GitLab [release](https://docs.github.com/en/repositories/releasing-projects-on-github/about-releases).
+1. Includes an R package at the top level whose package name is the same as the repository name.
+1. Is listed in the `URL` field of the corresponding CRAN page (if a package with the same name is on CRAN).
+
+and if the R package:
+
+1. Includes a license from the "Recommended licenses" section at the end of this policy.
+1. Does not have an advisory in the [R Consortium Advisory Database](https://github.com/RConsortium/r-advisory-database).
+1. Is not part of the CRAN mirror at <https://github.com/cran>.
+
+## Manual review
+
+R-multiverse [moderators](governance.md#moderator) review [pull requests](https://github.com/r-multiverse/contributions/pulls) that the bot flags for manual review.
+The [moderator](governance.md#moderator) inspects the package for compliance with R-multiverse policies, including but not limited to [Acceptable Use](aup.md), [Terms of Use](terms.md), and [Code of Conduct](conduct.md).
+The moderator accepts the contribution if and only if it complies with all policies.
+
+## Removal
+
+R-multiverse may remove a package from its own repositories at any time if the package violates R-multiverse policies.
+
+## Modification
+
+No R-multiverse staff member (administrator, moderator, or otherwise) may modify a package in R-multiverse without the explicit consent of the owners declared in the license of the package.
+
+## Recommended licenses
+
+The [Acceptable Use Policy](aup.md) prohibits packages that "violate any applicable laws, regulations, or third-party rights, including intellectual property rights".
+In practice, this means each package must have a valid open-source license.
+The following is a list of valid open-source licenses that the bot automatically accepts during reviews.
+If the package includes one of the license below, it implies that the authors consent to distribute the package in R-multiverse.
```

Comment:
```suggestion
```

Reply:
If anyone objects to their package being distributed on R-multiverse, we can point to the license, and/or we can invite them to submit a PR to r-multiverse/contributions to have it removed from community.